apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.tsx

```diff
@@ -54,14 +57,9 @@ const BillingBreakdown = () => {
 
         {invoiceFeatureEnabled && (
           <>
-            <p className="text-sm">Upcoming cost for next invoice</p>
             <p className="text-sm text-foreground-light">
-              The following table shows your upcoming costs excluding credits. Depending on your
-              usage, the final amount may vary. Next invoice on{' '}
-              <span className="text-foreground-light whitespace-nowrap">
-                {billingCycleEnd.format('MMM DD, YYYY')}
-              </span>
-              . See{' '}
+              The table shows your upcoming invoice, excluding credits. Depending on your usage, the
+              final amount may vary. See{' '}
```

Comment:
Now that I'm thinking about it again, I'm not sure whether it's a good idea to say that this is the customer's upcoming invoice. That has some final, binding aspect to it which is simply not the case given that the final amounts may vary (not sure whether the hint regarding varying amounts actually helps)